piv/pcsc_unix.go

```diff
@@ -18,14 +18,11 @@ package piv
 
 // https://ludovicrousseau.blogspot.com/2010/04/pcsc-sample-in-c.html
 
-// TODO: Figure out if linux flags should use pkg-config instead
-
 // #cgo darwin LDFLAGS: -framework PCSC
-// #cgo linux CFLAGS: -I/usr/include/PCSC
-// #cgo linux LDFLAGS: -lpcsclite
-// #cgo freebsd CFLAGS: -I/usr/local/include/
-// #cgo freebsd CFLAGS: -I/usr/local/include/PCSC
-// #cgo freebsd LDFLAGS: -L/usr/local/lib/
+// #cgo pkg-config: libpcsclite
```

Comment:
```suggestion
// #cgo linux pkg-config: libpcsclite
```
I have tested this change on linux and macOS and it still compiles correctly on linux while preventing the errors I was getting on macOS.